make.js

```diff
@@ -228,16 +234,20 @@ target.build = function() {
                 }
             });
 
-            // npm install the common module to the task dir
-            cd(taskPath);
-            modPackFiles.forEach(function(modPackFile) {
-                run(`npm install --save-exact file://../../_build/Tasks/Common/${modPackFile}`);
-            })
+            // npm install the common modules to the task dir
+            if (modPackFiles.length) {
+                cd(taskPath);
+                var modPackPaths = modPackFiles.map(function (val) {
+                    return `file:../../_build/Tasks/Common/${val}`;
+                });
+                run(`npm install --no-save ${modPackPaths.join(' ')}`);
+                run('npm dedupe');
+            }
         }
 
         // build Node task
         if (shouldBuildNode) {
-            buildNodeTask(taskPath, outDir);
+            buildNodeTask(taskPath, outDir, /*skipNpm*/ true);
```

Comment:
the npm install has already been done, dont do it again or it will wipe out the common modules since they weren't saved into the package.json